sysdump/sysdump.go

```diff
@@ -162,7 +165,7 @@ func (c *Collector) Run() error {
 			Description: "Collect Kubernetes nodes",
 			Quick:       true,
 			Task: func(_ context.Context) error {
-				if err := writeYaml(absoluteTempPath(kubernetesNodesFileName), n); err != nil {
+				if err := writeYaml(c.AbsoluteTempPath(kubernetesNodesFileName), n); err != nil {
```

Comment:
While at it, instead of having all these `writeYaml(c.AbsoluteTempPath(...))` calls could we define `(*Collector).writeYAML` as follows and then call that?

```go
func (c *Collector) writeYAML(p path, o runtime.Object) error {
	return writeYaml(c.AbsoluteTempPath(p), o)
}
```